src/Workspace/WorkspaceItem.elm

```diff
@@ -300,13 +306,20 @@ viewItem closeMsg toOpenReferenceMsg toUpdateZoomMsg ref item zoomLevel isFocuse
                 ref
                 attrs
                 (viewNames docZoomMsg detail.info (Category.Type Type.AbilityType))
-                [ ( UI.nothing, viewBuiltin item )
-                , viewSource toOpenReferenceMsg item
+                [ ( UI.nothing, viewBuiltin data.item )
+                , viewSource toOpenReferenceMsg data.item
                 ]
 
 
-view : msg -> (Reference -> msg) -> (Zoom -> msg) -> WorkspaceItem -> Bool -> Html msg
-view closeMsg toOpenReferenceMsg toUpdateZoomMsg workspaceItem isFocused =
+view :
+    msg
+    -> (Reference -> msg)
+    -> (Zoom -> msg)
+    -> (Id Doc -> msg)
```

Comment:
Interactivity on this level is getting a little bad—will likely need to be made better next time we need another msg.